app/src/main/java/com/hedvig/app/feature/onbarding/ui/ChoosePlanActivity.kt

```diff
@@ -0,0 +1,131 @@
+package com.hedvig.app.feature.onbarding.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.core.view.updatePaddingRelative
+import com.hedvig.android.owldroid.graphql.ChoosePlanQuery
+import com.hedvig.android.owldroid.type.EmbarkStoryType
+import com.hedvig.app.BaseActivity
+import com.hedvig.app.R
+import com.hedvig.app.databinding.ActivityChoosePlanBinding
+import com.hedvig.app.feature.marketpicker.MarketProvider
+import com.hedvig.app.feature.onbarding.ChoosePlanViewModel
+import com.hedvig.app.feature.onbarding.OnboardingModel
+import com.hedvig.app.feature.webonboarding.WebOnboardingActivity
+import com.hedvig.app.ui.animator.ViewHolderReusingDefaultItemAnimator
+import com.hedvig.app.util.extensions.view.remove
+import com.hedvig.app.util.extensions.view.setHapticClickListener
+import com.hedvig.app.util.extensions.view.show
+import com.hedvig.app.util.extensions.view.updateMargin
+import com.hedvig.app.util.extensions.viewBinding
+import d
+import dev.chrisbanes.insetter.doOnApplyWindowInsets
+import dev.chrisbanes.insetter.setEdgeToEdgeSystemUiFlags
+import org.koin.android.ext.android.inject
+import org.koin.android.viewmodel.ext.android.viewModel
+
+class ChoosePlanActivity : BaseActivity(R.layout.activity_choose_plan) {
+    private val binding by viewBinding(ActivityChoosePlanBinding::bind)
+    private val marketProvider: MarketProvider by inject()
+    private val viewModel: ChoosePlanViewModel by viewModel()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding.apply {
+            root.setEdgeToEdgeSystemUiFlags(true)
+            toolbar.doOnApplyWindowInsets { view, insets, initialState ->
+                view.updatePaddingRelative(top = initialState.paddings.top + insets.systemWindowInsetTop)
+            }
+            continueButton.doOnApplyWindowInsets { view, insets, initialState ->
+                view.updateMargin(bottom = initialState.margins.bottom + insets.systemWindowInsetBottom)
+            }
+            recycler.doOnApplyWindowInsets { view, insets, initialState ->
+                view.updatePaddingRelative(bottom = initialState.paddings.bottom + insets.systemWindowInsetBottom)
+            }
+            setSupportActionBar(toolbar)
+            supportActionBar?.setDisplayHomeAsUpEnabled(true)
+            supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_back)
```

Comment:
```suggestion
```

Replace these with the `MaterialToolbar`-equivalents 🙂 https://material.io/develop/android/components/app-bars-top